apm-agent-plugins/apm-grpc/apm-grpc-test-latest/src/test/java/co/elastic/apm/agent/grpc/latest/testapp/generated/HelloGrpc.java

```diff
@@ -42,7 +42,7 @@
 /**
  */
 @javax.annotation.Generated(
-    value = "by gRPC proto compiler (version 1.29.0)",
+    value = "by gRPC proto compiler (version 1.30.0)",
```

Comment:
Why is that related?

Reply:
It's changed after running tests, I think. Now in the master branch the value is already "1.30.0"